src/CoreBundle/Twig/Node/DeprecatedTemplateNode.php

```diff
@@ -11,29 +11,17 @@
 
 namespace Sonata\CoreBundle\Twig\Node;
 
-use Twig\Compiler;
-use Twig\Node\Expression\AbstractExpression;
-use Twig\Node\Node;
+@trigger_error(
+    'The '.__NAMESPACE__.'\TemplateBoxNode class is deprecated since version 3.x and will be removed in 4.0.'
+    .' Use Sonata\Twig\Node\TemplateBoxNode\FlashManager instead.',
```

Comment:
FlashManager again, also you mention `TemplateBoxNode` while in extends there isn't that part of the namespace